Error in build in openSUSE Open Build Service

## Lapce Version
0.2.0+git321 (commit d5560f8) 

Notes: Vendored dependencies are cargo updated before vendored

## System information

OpenSUSE Tumbleweed

## Relevant Build Log

```
[  868s]    Compiling lapce-proxy v0.2.0 (/home/abuild/rpmbuild/BUILD/lapce-0.2.0+git321/lapce-proxy)
[  872s]    Compiling lapce-data v0.2.0 (/home/abuild/rpmbuild/BUILD/lapce-0.2.0+git321/lapce-data)
[  874s] error[E0106]: missing lifetime specifier
[  874s]   --> lapce-data/src/config.rs:38:27
[  874s]    |
[  874s] 38 |     pub const LAPCE_WARN: &str = "lapce.warn";
[  874s]    |                           ^ expected named lifetime parameter
[  874s]    |
[  874s] help: consider using the `'static` lifetime
[  874s]    |
[  874s] 38 |     pub const LAPCE_WARN: &'static str = "lapce.warn";
[  874s]    |                            +++++++
[  874s] help: consider introducing a named lifetime parameter
[  874s]    |
[  874s] 37 ~ impl<'a> LapceTheme {
[  874s] 38 ~     pub const LAPCE_WARN: &'a str = "lapce.warn";
```

Full log found in this link: https://build.opensuse.org/package/live_build_log/editors/lapce/openSUSE_Tumbleweed/x86_64

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,7 +3,7 @@ name = "lapce"
 version = "0.2.1"
 authors = ["Dongdong Zhou <dzhou121@gmail.com>"]
 edition = "2021"
-rust-version = "1.62"
+rust-version = "1.64"
 default-run = "lapce"
 
 [dependencies]

